scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2008. All rights reserved.
+#
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+CHAIN=RH-Firewall-1-INPUT
+OP=$1
+rule="RH-Firewall-1-INPUT -p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+if [[ "${OP}" == "start" ]]; then
```

Comment:
It would be more idiomatic to use `case`:

```
case "${OP}" in
	start)
	...
	;;
	stop)
	...
	;;
	*)
	echo "usage .."
	exit 1
	;;
esac
exit 0